diff --git a/tasks/bin/cross-ruby.rake b/tasks/bin/cross-ruby.rake
index 8317a2a3..8ed21718 100644
--- a/tasks/bin/cross-ruby.rake
+++ b/tasks/bin/cross-ruby.rake
@@ -116,11 +116,30 @@
         "--host=#{mingw_host}",
         "--target=#{mingw_target}",
         "--build=#{RUBY_BUILD}",
-        '--enable-shared',
+        '--enable-install-static-library',
+        '--disable-jit-support',
+        'ac_cv_lib_z_uncompress=no',
+        'ac_cv_lib_crypt_crypt=no',
+        'ac_cv_func_crypt_r=no',
         '--disable-install-doc',
         '--with-ext=',
       ]
 
+      if mingw_host =~ /darwin/
+        options += [
+          '--enable-static',
+          '--disable-shared',
+        ]
+      else
+        options += [
+          '--enable-static',
+          '--enable-shared',
+        ]
+      end
+
+      # https://github.com/docker-library/ruby/issues/308
+      options << "--with-coroutine=arm32" if major == "2.7" && mingw_target =~ /arm-linux-musl/
+
       # Force Winsock2 for Ruby 1.8, 1.9 defaults to it
       options << "--with-winsock2" if major == "1.8"
       options << "--prefix=#{install_dir}"
